### DISCUSSION TITLE
Thoughts on To-Do vs. QuikNote

### DISCUSSION THREAD

**Discussion by noa7777** [gsantner/markor#2433] [Ideas]
To-Do and QuikNote, as productivity tools, are supposed to be fast.
But now, click on QuickNote, and then I see the page jumping crazy to the end. I don't think it's fast enough! I'd rather recommend that you don't jump to the end and start with the first line

**Comment by gsantner** [comment]
When you write new text to your real paper notebook, do you open it and turn pages till where you left off last time - to add new stuff?
Or do you insert new stuff at the first page constantly? 

Markors QuickNote and ToDo are focused on quick adding stuff and taking further care/splitoff when you have time to.

If you think its not fast enough, a fancy loading animation could be in place like other apps to simulate something being fast. Loading and file and rendering is not instant. But I think it's good enough and Markor don't need to hide behind animation whats going on.

**Reply by noa7777** [reply]
What I'd like to say is that you can add a setting where you click Edit and don't jump to the last line, because you use QuikNote to record your thoughts, and if you record them to the end of the article, you need to scroll to the end every time you look at the most recent ideas (in other markdon editors). It's a bit bad!

**Comment by harshad1** [comment]
I have a 600 line quicknote (yes it is in desparate need of being processed) and it opens at the bottom nearly instantly. Faster than pretty much anything else **including even the google search from my launcher**

If it seems too slow please make a recording to show so this can be looked at.

**Reply by noa7777** [reply]
QuickNote in Markor
When you enter this page, you will slide down the page crazily, slide the page crazily, slide the page crazily, until the last line of the article, so that I personally feel more "dizzy". It's a waste of time, like if I have a good idea and then scroll down the page like crazy, it's slower than just recording it on the first line!

**Reply by harshad1** [reply]
Are you on a recent version?

And do you have bigger headings enabled?

**Comment by gsantner** [comment]
Then just use a normal file, where there is no special behaviour? problem solved. 

Not everything fits for everyone.
